src/util/cast.js

```diff
@@ -1,13 +1,5 @@
 const Color = require('../util/color');
 
-/**
- * Store and possibly polyfill Number.isNaN. Number.isNaN can save time over
- * self.isNaN by not coercing its input. We need to polyfill it to support
- * Internet Explorer.
- * @const
- */
-const _NumberIsNaN = Number.isNaN || isNaN;
```

Comment:
@mzgoddard, do you know why we added this change? I thought I might have commented on it in the original PR, but I think my comment was probably about something else...

Reply:
Relevant discussion is here: https://github.com/LLK/scratch-vm/pull/1709/files/5b10d41ba32f321a94aac4a6fb39656cfca4a2f0#diff-70e40f35658352341bf64fe5c0c23303

Seems it was swapped from `isNaN` to `Number.isNaN` for performance benefit, and there was some discussion about that. The only mention of IE is from mzgoddard:

> I made the change (with back compat support in mind for Internet Explorer which does not provide Number.isNaN, in which case self.isNaN is a safe backup) and recorded some new numbers.

Reply:
@kchadha @towerofnix Yeah I don't recall the reason for the polyfill addition. I think it was because Number.isNaN was a new "idea" for the Scratch repos. self.isNaN is used the rest of the time. (We should use Number.isNaN instead though.)